packages/coding-standards/easy-coding-standard.yml

```diff
@@ -1,6 +1,9 @@
 imports:
     # for PSR-2 description, see: http://www.php-fig.org/psr/psr-2/
     - { resource: '%vendor_dir%/symplify/easy-coding-standard/config/psr2.yml' }
+    # some checkers use BetterPhpdocParser services which must be configured (file renamed in symplify/better-phpdoc-parser v5.4.14)
+    - { resource: '%vendor_dir%/symplify/better-phpdoc-parser/config/config.yml', ignore_errors: true }
+    - { resource: '%vendor_dir%/symplify/better-phpdoc-parser/config/config.yaml', ignore_errors: true }
```

Comment:
what about just one line with `config.y*ml` and maybe no ignore_error would be needed

Reply:
I haven't thought of that, that seem slightly less readable to me, but maybe it's better... I would rather use `config.y{,a}ml` or `config.{yml,yaml}` than `config.y*ml` to be more explicit.

To be honest, I'm slightly more inclined to leave it as-is in sake of readability. What do you think?